rtl/include/eclhelper.hpp

```diff
@@ -1486,13 +1486,24 @@ struct IHThorRollupArg : public IHThorArg
     virtual size32_t transform(ARowBuilder & rowBuilder, const void * _left, const void * _right) = 0;
 };
 
+enum
+{
+    HFDwholerecord  = 0x0001,
```

Comment:
Should rename to match the prefix of the other dedup flags.  (Should have been HDF anyway).